_data/navigation/en_US.yml

```diff
@@ -64,6 +64,9 @@ sidebar_pages:
   -
     title: Seedminer
     url: seedminer
+  -
```

Comment:
Add a space between Seedminer and (TWN)
title -> Seedminer (WTN)
url -> seedminer-(twn)